grails-app/domain/org/broadinstitute/orsp/IssueExtraProperty.groovy

```diff
@@ -27,6 +27,8 @@ class IssueExtraProperty {
     public static final String PM = "pm"
     public static final String SOURCE = "source"
     public static final String ACTOR = "actor"
+    public static final String PROJECT_QUESTIONARE = "questions"
+    public static final String PROJECT_TITLE = "pTitle"
```

Comment:
Prefer longer descriptions here. This value gets saved as the name in the issue extra property table so having a better value will make it easier to understand when looking at the database directly.
```suggestion
    public static final String PROJECT_TITLE = "projectTitle"
```